assets/components/AuditInformation.jsx

```diff
@@ -28,7 +28,7 @@ class AuditInformation extends React.Component {
         const users = get(this.props, 'users.length') ? this.props.users : this.props.editUsers;
 
         if (field === '_created' && item._created) {
-            const creator = getItemFromArray(item.original_creator,
+            const creator = getItemFromArray(item.users,
```

Comment:
why this change?

Reply:
This was done earlier when `original_creator` was not updated when we created topics via API. Since now it is updated as per the user, this is not required.